src/warnet/plugins.py

```diff
@@ -0,0 +1,218 @@
+import copy
+import importlib.util
+import inspect
+import os
+import sys
+import tempfile
+from pathlib import Path
+from types import ModuleType
+from typing import Optional
+
+import click
+import inquirer
+import yaml
+from inquirer.themes import GreenPassion
+
+from warnet.constants import (
+    CONTAINER_TAG,
+    ENABLED_TAG,
+    MISSION_TAG,
+    PLUGIN_YAML,
+    PLUGINS_TAG,
+    USER_DIR_TAG,
+    WARNET_USER_DIR_ENV_VAR,
+)
+
+
+class PluginError(Exception):
+    pass
+
+
+imported_modules: dict[str, ModuleType] = {}
+
+
+@click.group(name=PLUGINS_TAG)
+def plugins():
+    """Control plugins"""
+    pass
+
+
+@plugins.command()
+@click.pass_context
+def ls(ctx):
```

Comment:
we took out "scenarios list" since the available scenarios is up to the user and could be files from anywhere, I dont think we need to list plugins

Reply:
Now gone.